### ISSUE TITLE
Participation fees (bid document and submission costs)

### ISSUE THREAD

**Issue by practicalparticipation** [open-contracting/standard#33]
A participation fees extension is under consideration for version 1.1 of OCDS based on the discussion in this thread.
## The issue

Requirement generated from [User Research process](https://docs.google.com/spreadsheets/d/1m_Kr7I5XqM7PFcz75ZNmcdFCzLaike_IsEmAh5eN-WE/edit#gid=2035729003): R6.

Users want to know the **price of tender documentation**. 

Whilst Open Contracting principles would encourage governments to make tender documents available free of charge, this is not always the case. Sometimes documents are only available on application or on payment of a fee.

Sometimes a fee is required to submit documents
## Proposal

We introduce a `participationFee` extension to `tender` which provides for:
- feeRequired - true/false
- feeType - document or participation
- fee - value of the fee (currency and amount
## Engagement

Please indicate support or opposition for this proposal using the +1 / -1 buttons or a comment. If opposing the proposal, please give clear justifications, and where possible, make an alternative proposals.

**Comment by joshuacpowell** [comment]
Running into this in Vietnam, as well. I would suggest it may be frequent enough to revisit the idea of incorporating into the standard.

**Comment by timgdavies** [comment]
It would be useful to get examples of any existing supply-side data including tender pricing information.

@joshuacpowell Is there an example from Vietnam of the structure of data that is held?

We need to track down the example from Nepal Lindsey mentions and include it in this thread.

**Comment by joshuacpowell** [comment]
Based on Vietnam, I would suggest perhaps 3 fields
1) Fee required Y/N?
2) Fee type (participation or bid doc acquisition)
NOTE: This is because in some cases/countries you can acquire the bid docs for free, but must pay a fee in order to submit a bid, while in other countries you must pay just to acquire the bid docs
3) Amount of fee (currency, amount)

You could also consider adding payment method as a 4th (i.e. is online payment available), but this may be more detail than makes sense for the standard globally.

**Comment by duncandewhurst** [comment]
I believe [this](http://eproc.dor.gov.np/tender_details.php?tid=51282) is the example from Nepal.

The bid documents are available to download free of charge at the bottom of the page but the fee must be paid for submission (either electronic or hard copy).

**Comment by LindseyAM** [comment]
There is a difference between bid submission fee and price of bid docs. In either case, this seems like an extension

**Comment by timgdavies** [comment]
I've updated the initial post here with a proposal based on @joshuacpowell's suggestion of a three-field object.

**Comment by myroslav** [comment]
I'm somewhat puzzled of how extension should work if there is both participation and document fee. Should we split this extension on two independent ones? One for documents and another one for participation. They have completely different cases. And while in ProZorro we have free and open tender documentation, participation is not free. This we'd be interested in one extension and completely ignore the other one. The only common this in the proposals is that in both there is some fee involved.

**Comment by timgdavies** [comment]
This is a good point. We could make this an array of `participationFees` where each entry just consists of:
- feeType
- feeValue

So that if you have just participation fees you would have:

```
"tender": {
    "participationFees": [
      {
          "type":"participation",
          "value": {
               "currency":"GBP",
               "amount":10.00
           } 
    ]
}
```

and if you had both document access and participation fees you might have:

```
"tender": {
    "participationFees": [
      {
          "type":"document",
          "value": {
               "currency":"GBP",
               "amount":8.00
           } 
      {
          "type":"participation",
          "value": {
               "currency":"GBP",
               "amount":10.00
           } 
    ]
}
```

With the logic being that unless fees are declared for category of interaction with the tender, they should be assumed to be zero.

**Comment by timgdavies** [comment]
The proposal was discussed on today's OCDS community call. There was general support for this approach.

**Comment by chrisalexsmith** [comment]
There might be a third type of participation fee - a win fee payable by the successful tenderer.

**Comment by duncandewhurst** [comment]
There is a draft extension, based on the modelling proposed by Tim, [here](https://github.com/open-contracting/ocds_upgrade_33).

The extension implements a codelist with the following codes:

* document - a fee payable for access to bidding documents
* submission - a fee payable for the submission of bids
* win - a fee payable by the winning bidder

I've used 'submission' rather than 'participation' as a code to avoid ambiguity with the name of the property.

Assigning to @ekoner to review

**Comment by timgdavies** [comment]
I've been reviewing the World Bank Benchmarking Public Procurement Report and that points to another category we may need to consider:

* 'Bid security deposits' which bidders may need to pay, but which are refunded after the bid process. 

I suggest we consider a 'deposit' codelist entry. 

Looking at [an example for Afhanistan](http://bpp.worldbank.org/data/exploreeconomies/afghanistan/2017#bpp_plc) it appears these securities can be provided in a number of forms, but I would propose we stick to only recording cash value in OCDS. However, we may want to include a ```description``` field for information on the nature of the fee to optionally be supplied. 

On the fees payable by the winning bidder, I've been looking to find examples of this. They best I could locate was FAQs about whether people need to pay to be included on a framework (where the answer in the FAQs I found was 'no', but I'm thinking implies there may be cases where the answer is 'yes'). 

@chriscrownagents Any other pointers to examples of 'winner pays' policies would be much appreciated.

**Comment by timgdavies** [comment]
@duncandewhurst I've just submitted a pull request that makes some small documentation tweaks and adds 'deposits'.

I'm also left wondering whether we need to include a distinction between:

* The fee to access documents **at all*
* The fee to access hard copy / official copies of documents

This may help make the case discussed in [these usage notes](https://github.com/open-contracting/ocds_upgrade_33/tree/td-suggested-changes#usage-notes) less confusing. I.e. when you can get documents for free (useful for civil society monitors to know), but you need to buy an official copy before you can submit into the process.

**Comment by andrewlorien** [comment]
In AusTender there are never any fees for documents or submissions, but in NSW eTendering, there is an admin fee to download the documents, or have a hard copy posted to you (it's not an "official copy", just a paper copy).  The two fees are different, and set by the buyer per tender.  It is intended to be a nominal admin fee.  As you'd expect, the download fee is very rarely used, but the hardcopy fee is quite common.

**Comment by ekoner** [comment]
Reviewed: Now ready for peer review.

[Sample JSON](https://github.com/open-contracting/sample-data/tree/ocds_upgrade_33/fictional-example)

**Comment by timgdavies** [comment]
Included as a core extension in 1.1: http://standard.open-contracting.org/latest/en/extensions/participation_fee/
